packages/react/src/components/Alert/Alert.tsx

```diff
@@ -87,6 +99,9 @@ export interface AlertProps
 
   /** An alert may be formatted to display a warning message. */
   warning?: boolean
+
+  /** Id of a wrapper containing header and content */
```

Comment:
consider renaming wrapper to body, we use wrapper in other components for wrapping the whole component (root)